src/stylesheets/common/_variables.scss

```diff
@@ -17,7 +17,7 @@
 
 // Display type used for headings, logo, etc.
 $display-font-family: 'Poppins', 'Open Sans', Helvetica, Arial, sans-serif;
-$display-font-weight-bold: 600;
+$display-font-weight-bold: 700;
```

Comment:
Oh, as long as it was intentional.  I was worried it was a holdover from all the previous font changes.